src/metabase/pulse/render.clj

```diff
@@ -153,5 +153,6 @@
 
 (defn render-pulse-card-to-png
   "Render a `pulse-card` as a PNG. `data` is the `:data` from a QP result (I think...)"
-  ^bytes [timezone pulse-card result]
-  (png/render-html-to-png (render-pulse-card :inline timezone pulse-card result) card-width))
+  ^bytes [timezone-id pulse-card result]
+  {:pre [((some-fn nil? string?) timezone-id)]}
```

Comment:
Why `:pre` rather than Schema, or more generally do we have some guidelines when to use one over the other?

Reply:
That's a good question we should probably be using schema here. I'll change it